app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -61,6 +60,7 @@ const val STATE_FRAGMENT_STORY_ID_ARGUMENT_KEY = "StateFragmentPresenter.state_f
 const val STATE_FRAGMENT_EXPLORATION_ID_ARGUMENT_KEY =
   "StateFragmentPresenter.state_fragment_exploration_id"
 private const val TAG_AUDIO_FRAGMENT = "AUDIO_FRAGMENT"
+const val STATE_FRAGMENT_RAW_USER_ANSWER_KEY = "StateFragmentPresenter.raw_user_answer"
```

Comment:
Why define this here if it's only used in StateFragment? Suggest moving it there & making it private.

Reply:
Moved to StateFragment.kt